### DISCUSSION TITLE
Problem with webhook

### DISCUSSION THREAD

**Discussion by Ramxnchv** [swellstores/community#246] [Help]
Hello, I had a bug a few days ago in a webhook response and now fixed it, but my new webhooks never seem to fire (always set to "pending": true, "attempts": 0 and on /events "webhooks_pending": 1). I'm a bit lost on how can I reset my webhooks or get them working again, maybe the failing webhook with 63 attempts (which i tried to delete to solve the problem) is still blocking the rest. Thanks for your help!

![image](https://github.com/swellstores/community/assets/38292919/2b4f18a8-8b4e-48ec-891d-32d642c984b8)

**Comment by Ramxnchv** [comment]
The bug was still happening sometimes and thats why webhooks were not working, sorry 😅

**Reply by ericingram** [reply]
We are seeing delays in webhook processing currently. Not sure if it’s related though.
